src/core/traits/scalar.rs

```diff
@@ -278,8 +278,7 @@ impl FloatEx for f64 {
     }
     #[inline(always)]
     fn acos_approx(self) -> Self {
-        // TODO: clamp range
-        f64::acos(self)
+        f64::acos(self.clamp(-1.0, 1.0))
```

Comment:
`clamp` requires 1.50.0. Probably easier to use `self.max(-1.0).min(1.0)` instead for now.